CMakeLists.txt

```diff
@@ -182,6 +188,33 @@ else (USE_PACKAGE_MANAGER)
 
 endif (USE_PACKAGE_MANAGER)
 
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
+  SET(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /DEBUG:FULL /Od")
+  SET(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} /DEBUG:FULL /O2")
+  SET(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /O2")
+  if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    add_compile_definitions(DEBUG)
+  elseif(CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
+    add_compile_definitions(DEBUG)
+  endif()
+  link_libraries(dbghelp.lib)
```

Comment:
Why do you mess with CMake Flags here ?
Why do you override optimization / debug here?

There is a seperate cmake file for CompileOptions:
https://github.com/CytopiaTeam/Cytopia/blob/master/cmake/CompileOptions.cmake

Reply:
Are we using the CompileOptions file at all? I thought we weren't. Also, I need lines 195-199 to add the DEBUG directive so that we don't print debug logs for Release

Reply:
Ok so it looks like we are using them

Reply:
done